### ISSUE TITLE
[BUG] - Importing postman collection results in empty collection

### ISSUE THREAD

**Issue by kobenguyent** [flawiddsouza/Restfox#234]
When trying to import a postman collection, restfox just results in empty collection

![Screenshot 2024-09-05 at 16 22 49](https://github.com/user-attachments/assets/3b815b4a-9cac-461f-82a3-a569e80a86cf)

It works fine with Insomnia

![Screenshot 2024-09-05 at 16 24 37](https://github.com/user-attachments/assets/05381a78-4cde-485f-9d68-6ad1c756ac0f)

Postman collection
[Argos API.postman_collection.json](https://github.com/user-attachments/files/16895149/Argos.API.postman_collection.json)

**Comment by flawiddsouza** [comment]
Hey @kobenguyent, looks like import from postman works fine before this commit: 

https://github.com/flawiddsouza/Restfox/commit/bd6c5a22fd2383581721834a26bb56f8bb8f7af9: feat(ui): Import > import scripts from postman collections (#218)

I did
```
git checkout bd6c5a22fd2383581721834a26bb56f8bb8f7af9^
```

And it worked correctly.

Then I did
```
git checkout bd6c5a22fd2383581721834a26bb56f8bb8f7af9
```

And it didn't work. So the issue was introduced here. I'm checking this code now.
